CHANGES.rst

```diff
@@ -18,6 +18,10 @@ Version 3.1.0 (2019-XX-XX)
 - fix ``_apply_partition_key_predicates`` ``FutureWarning``
 - serialize :class:`~kartothek.core.index.ExplicitSecondaryIndex` to parquet
 - improve messages for schema violation errors
+- Ensure binary column names are read as type ``str``:
```

Comment:
Please use `-` for the nested list and add an empty line between between the parent and the nested list. Try to build the docs to see why this is required.